org.devloop.Do.json

```
@@ -0,0 +1,37 @@
+{
+    "app-id" : "org.devloop.Do",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "todo",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/rust-flatpak/cargo"
```

```suggestion
            "CARGO_HOME" : "/run/build/todo/cargo"
```

It has to be the build dir for the module